spec/lib/services/account-api-service-spec.js

```diff
@@ -57,98 +70,193 @@ describe("Account API Service", function() {
+                expect(user.username).to.equal('admin');
                 expect(user.password).to.be.an('undefined');
-                expect(user.role).to.be.a('string');
+                expect(user.role).to.equal('Administrator');
                 expect(user.privileges).to.be.instanceof(Array);
+                expect(user.privileges[0]).to.equal('ConfigureUsers');
             });
         });
-    });
 
-    it('should get a user by name', function() {
-        waterline.localusers.findOne.resolves(user1);
-        return accountService.getUserByName('admin')
-        .then(function(user) {
-            expect(waterline.localusers.findOne).to.have.been.called;
-            expect(user.username).to.equal('admin');
-            expect(user.password).to.be.an('undefined');
-            expect(user.role).to.equal('Administrator');
-            expect(user.privileges).to.be.instanceof(Array);
-            expect(user.privileges[0]).to.equal('ConfigureUsers');
+        it('should create a user', function () {
+            waterline.localusers.findOne
+                .onFirstCall().resolves()
+                .onSecondCall().resolves(user1);
+            waterline.localusers.create.resolves(user1);
+            return accountService.createUser(user1)
+            .then(function (user) {
+                expect(waterline.localusers.findOne).to.have.been.called;
+                expect(waterline.localusers.create).to.have.been.called;
+                expect(user.username).to.equal('admin');
+                expect(user.password).to.be.an('undefined');
+                expect(user.role).to.equal('Administrator');
+                expect(user.privileges).to.be.instanceof(Array);
+                expect(user.privileges[0]).to.equal('ConfigureUsers');
+            });
         });
-    });
 
-    it('should create a user', function() {
-        waterline.localusers.findOne
-            .onFirstCall().resolves()
-            .onSecondCall().resolves(user1);
-        waterline.localusers.create.resolves(user1);
-        return accountService.createUser(user1)
-        .then(function(user) {
-            expect(waterline.localusers.findOne).to.have.been.called;
-            expect(waterline.localusers.create).to.have.been.called;
-            expect(user.username).to.equal('admin');
-            expect(user.password).to.be.an('undefined');
-            expect(user.role).to.equal('Administrator');
-            expect(user.privileges).to.be.instanceof(Array);
-            expect(user.privileges[0]).to.equal('ConfigureUsers');
+        it('should recreate a user', function () {
+            waterline.localusers.findOne
+                .onFirstCall().resolves(user1)
+                .onSecondCall().resolves(user1);
+            waterline.localusers.create.resolves(user1);
+            return accountService.createUser(user1)
+            .then(function () {
+                expect(false);
```

Comment:
@keedya : the existing code is not always correct. If we found some minor bug and can fix it with little effort, I strongly recommend to do this. Otherwise, we may forget it, and it's not worth to open a PR even an issue to track this.